apis/v1alpha1/httproute_types.go

```diff
@@ -319,8 +339,7 @@ type HTTPRouteFilter struct {
 	// types to extend the core API with implementation-specific behavior.
 	//
 	// +unionDiscriminator
-	// +kubebuilder:validation:Required
-	// +required
+	// +kubebuilder:validation:MaxLength=100
```

Comment:
Should `// +kubebuilder:validation:MinLength=1` be set?

Reply:
I'm wondering about this in other places as well. The field is required so there should be something in the field. Does the validation allow for empty string like `"  "`?

Reply:
Add minLength=1 validation, we likely need that on more strings as well. Unfortunately I can't find a way to handle/trim whitespace with kubebuilder validation tags.

Reply:
Added minLength=1 validation, we likely need that on more strings as well. Unfortunately I can't find a way to handle/trim whitespace with kubebuilder validation tags.